### PR TITLE
Docs for SR14

### PR DESCRIPTION
**Pull request by willgearty**
This is a placeholder for the docs for Stable Release 14, planned for release in June 2022. Feel free to add to it as you see fit.

### CODE REVIEW

File: docs/admin/releases/14/README.rst [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- If there are any required items, the module is required during registration, and the form will require selections to be made for those required items before it can be submitted
+- Required items may have associated costs and/or options; this is now the supported method for asking students which kind of lunch/meal they would like (e.g., pizza vs. sandwich)
+- The numbers of students that have requested particular options can be seen on the program dashboard
+- The sibling discount form has also been added to this module (if enabled in the program settings) and the discount value now reflects the respective program setting
+- The Splash Info Module, which previously included the lunch options and sibling discount forms, has been completely removed
+- The name that students record for the sibling discount can be retrieved as a field in the arbitrary user list
+
+Program management changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Added an admin checkbox interface for the steps that are required for ensuring a program is completely setup
```
Comment (milescalabresi) [inline docs/admin/releases/14/README.rst:81]: *set up, not setup

File: docs/admin/releases/14/README.rst [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+- Added a user interface for adding and editing class categories and class flag types (/manage/categoriesandflags)
+- Added a user interface to approve and reject grade change requests on the userview page
+- Added a user interface to create, edit, and delete permissions for individual users (on what was previously the Deadline Management page)
+- Added a user interface to create, edit, and delete URL and email redirects (/manage/redirects)
+- Added a user interface to edit registration receipts on the program settings page
+- Added a user interface to change the sequence and requiredness of program modules
+- Added a user interface for adding and removing students to/from the student lottery
+- The link to the admin pages has been removed for all themes
+
```
Comment (milescalabresi) [inline docs/admin/releases/14/README.rst:120]: Are there links to these new /manage/* pages anywhere (such as from the QSD textbox on /manage/programs/ page?

**Review by milescalabresi — APPROVED** [review]
Looks good. Left some small comments

File: docs/admin/releases/14/README.rst (hunk 2 above)
Comment (willgearty) [reply docs/admin/releases/14/README.rst:120]: Of course! (and that isn't a QSD box anymore)
image

**Pull request merged**